src/components/CommentCard.jsx

```diff
@@ -0,0 +1,22 @@
+import dateConverter from "../utils/dateConverter";
+
+export default function CommentCard({ comment }) {
+    if (!comment) {
+        return <div>Ain't no comments here my dude, now go on, get!!</div>
+    }
```

Comment:
This currently isn't showing. The error message may be better in your comments list so it can render if the article has no comments. Also please use semantic HTML rather than wrap it in a div. A p tag would be nicer